### PR TITLE
Added OS generated files to .gitignore

### PR DESCRIPTION
**Pull request by TheGemDev**
added os generated files to .gitignore this should prevent adding these files in a Pull Request
# OS generated files #

.DS_Store?
._*
.Spotlight-V100
.Trashes
ehthumbs.db
Thumbs.db

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,10 @@
 .idea
 .vscode/ipch
 /newIDE/app/src/UI/Theme/**/*ThemeVariables.*
+.DS_Store
+.DS_Store?
```
Comment (4ian) [inline .gitignore:65]: Is this one really necessary? Let's keep just the first one.

File: .gitignore [hunk 2]
```diff
@@ -61,3 +61,10 @@
...
+._*
+.Spotlight-V100
```
Comment (4ian) [inline .gitignore:67]: I would rather avoid these things that could encompass too much files.

File: .gitignore [hunk 3]
```diff
@@ -61,3 +61,10 @@
...
+.Trashes
+ehthumbs.db
```
Comment (4ian) [inline .gitignore:69]: For which OS is this?

File: .gitignore (hunk 2 above)
Comment (TheGemDev) [reply .gitignore:67]: Okay

File: .gitignore (hunk 3 above)
Comment (TheGemDev) [reply .gitignore:69]: .Spotlight-V100
.Trashes
Are generated by apple macintosh, 
._* would be removed also ehthumbs.db

File: .gitignore (hunk 1 above)
Comment (TheGemDev) [reply .gitignore:65]: Okay

**Comment by 4ian** [comment]
Thanks! 👍

**Pull request merged**